### PR TITLE
#1156: POC - Repeater validation errors.

### LINKED ISSUE #1156: Repeater doesn't show red text (error message) under fields when validation fails
## Description

Issue based on discussion #975

### Steps to reproduce

Create repeater block and try to validate it's fields

### Expected result

If validation fails, there should be error message under the incorrectly filled fields

### Actual result

Shows "Your submission could not be validated, please fix and retry" message but the fields are not being marked

### Versions

Twill version: 2.5.1
Laravel version: 8.58.0
PHP version:  7.4.9
Database engine: InnoDB

### PR DESCRIPTION
**Pull request by haringsrob**
## Description

This is an initial draft on how to get repeater errors to show in the editor.

It feels a bit weird but it does provide a solution to the problem. I am open to suggestions on how to make this better.

## Related Issues

Fixes #1156

### CODE REVIEW

**Comment by haringsrob** [comment]
An alternative solution to this would be to store the validation errors with the fields in the store, then we do not have to drill down. But we have to take care on submit to remove that non required data again.

**Comment by haringsrob** [comment]
Screenshot 2022-01-08 at 09 58 26

File: frontend/js/components/Textfield.vue [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <template>
-  <a17-inputframe :error="error" :note="note" :label="label" :locale="locale" @localize="updateLocale" :size="size" :name="name" :label-for="uniqId" :required="required">
+  <a17-inputframe :custom-error-key="customErrorKey" :error="error" :note="note" :label="label" :locale="locale" @localize="updateLocale" :size="size" :name="name" :label-for="uniqId" :required="required">
```
Comment (ifox) [inline frontend/js/components/Textfield.vue:2]: @haringsrob let's add this to other fields

**Comment by ifox** [comment]
Closing in favor of #1396.

**Pull request closed without merging**